README.md

```diff
@@ -105,3 +105,22 @@ docker build -f my-model.Dockerfile -t my-model-inference .
 That's it! You can now push your image to your favorite registry or reference
 it locally in your Weaviate `docker-compose.yaml` using the docker tag
 `my-model-inference`.
+
+### M1 Support 
```

Comment:
M1 is only supported when run locally. We might be able to run this in docker when MPS support becomes official but at the moment the nightly builds automatically redirects docker linux images (even amd64v8 ones) to some older linux version